.gitignore

```diff
@@ -70,3 +70,6 @@ yarn-error.log*
 fake-v1-api/
 mdn-yari-*.tgz
 function.zip
+
+# tabnine
+.dccache
```

Comment:
What's this?

Reply:
cache file created by https://www.tabnine.com/

Reply:
I don't like this. I think it's unrelated to removing `babel-loader`. 
Besides, it's not a thing that applies to all users of Yari. This Tabnine is some optional third-party software that we don't actively support. 

If a developer wants to use Tabnine, good for them, but they should use *their* `~/.gitignore_global` or *their* `.git/info/exclude`. We can't keep adding every single IDEs' files. For example, we don't have `.vscode` in there.